www/index.html

```diff
@@ -78,7 +78,6 @@
 </script>
 
 <script src="boot.js"></script>
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Promise,Array.from,Array.prototype.find,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Math.sign,Intl.~locale.en,Intl.~locale.id,Intl.~locale.ru,Intl.~locale.fr&unknown=polyfill"></script>
```

Comment:
OH have actually opened an issue with them . I Would update this when they reply. Nonetheless the size is not that much . Barely noticeable.